benchmarking/runs/run_diversify_2_friends.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+from benchmarking.data.interfaces import MockStudentProviderSettings
+from benchmarking.data.simulated_data.mock_student_provider import MockStudentProvider
+from benchmarking.evaluations.metrics.average_gini_index import AverageGiniIndex
+from benchmarking.evaluations.metrics.num_requirements_satisfied import (
+    NumRequirementsSatisfied,
+)
+from benchmarking.evaluations.metrics.num_teams_meeting_requirements import (
+    NumTeamsMeetingRequirements,
+)
+from benchmarking.evaluations.scenarios.diversify_social_min_2_friends import (
+    DiversifySocialMin2Friends,
+)
+from benchmarking.simulation.simulation import Simulation
+from models.enums import ScenarioAttribute, Relationship
+
+
+def run_diversify_2_friends():
+    """
+    Goal: Run diversify gender scenario, measure average gini index
```

Comment:
These comments should be updated

nit; function name could be more descriptive too